### ISSUE TITLE
Rule disabled by default.

### ISSUE THREAD

**Issue by Torguen** [godotengine/godot-proposals#9900]
### Describe the project you are working on

Not importan here.

### Describe the problem or limitation you are having in your project

I think that 90% of users do not use this rule, however it appears activated by default, taking up space in the editor.
The annoyance is that you will have to hide it every time you start a new project or test.
I propose that the rule be set to disabled by default.

![Captura5](https://github.com/godotengine/godot-proposals/assets/19652075/750a1bca-92c4-462a-a2f2-ac5bfa108e95)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Disable the default rule to free up space in the editor and avoid the hassle of having to do it manually every time you start a new project. I would say that maybe 10% of users use it.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

...

### If this enhancement will not be used often, can it be worked around with a few lines of script?

I think it is not possible

### Is there a reason why this should be core and not an add-on in the asset library?

It seems to me that this should be implemented in the editor.

**Comment by AThousandShips** [comment]
See also (IMO more reasonable change):
* https://github.com/godotengine/godot-proposals/issues/8795

**Comment by Torguen** [comment]
Yes, it is more reasonable, the goal would be a way to hide it forever, even when a new project is generated.

**Comment by AThousandShips** [comment]
Yes, that's what that proposal proposed? It's an editor setting that's proposed, you have to turn it off once and it'll always be off for any project 

Because it's useful, and hiding it by default makes it hard to find

It doesn't take up any real amount of space except on tiny screens

**Comment by Torguen** [comment]
Who is it useful for? not for me, and, as I said, no one ever talks about it in any video, nor does they explain what it is used for... it doesn't seem very useful according to these facts.

I'm not saying it's completely useless, but it seems to me to have very little use among users.

Thanks for the information, I will look into that option.

**Comment by AThousandShips** [comment]
It's useful to measure things? That's what a ruler is for no?

You're basing your assumption on what people talk about in videos only? Or have you talked to users? I don't think your assumption is well founded

Again it takes up nearly no space at all, so I'd say it's more annoying for those who do use it to have to activate it every time than the tiny space it takes up

**Comment by Torguen** [comment]
I assume that if no one talks about it or explains what it is for or uses it in the tutorial, it is because they simply do not use it.

Isn't this proof that you don't get much use for that rule?
Of course you could talk to users to reinforce that information but I bet they will say that they don't use it.

I will tell you the same thing regarding the space that the rule occupies in the editor, have you asked users if the space it occupies is enough for them to prefer to have the rule disabled by default?

I will not discuss further, let the users decide!

**Comment by AThousandShips** [comment]
My simple explanation why tutorials don't necessarily talk about it is that it:
* Is obvious what it does and doesn't need explanation
* That most tutorials don't deal with *actual* design work as such but prepared things and has all the layouts and art prepared

But I respect your request not to discuss further beyond giving my explanation, but I don't think that is a very productive attitude

**Comment by Torguen** [comment]
Well, I won't argue any further because I have nothing more to say.

My proposal is based on the use I make of it (no use) and what I see in the videos, whether they are tutorials or not, I simply don't see anyone using the ruler.

**Comment by KoBeWi** [comment]
AFAIK rulers are currently the only way to add guides to your scene.

**Comment by AThousandShips** [comment]
> AFAIK rulers are currently the only way to add guides to your scene.

Can confirm this, and can't think of a reasonable way to change the UI to accommodate this without the rulers

**Comment by Kobberholm** [comment]
> Who is it useful for? not for me, and, as I said, no one ever talks about it in any video, nor does they explain what it is used for... it doesn't seem very useful according to these facts.
> 
> I'm not saying it's completely useless, but it seems to me to have very little use among users.

I just used the rulers extensively for the last couple of hours or so, debugging layout issues where a relatively complex tree of control nodes had to dynamically appear on top of and align with pre-existing Node2Ds.
I have no idea how I would have solved that without rulers, as it was dealing with nodes being 1 and 2 pixels off and I was zoomed in to 1000%.

As AThousandShips said, it is obvious how rulers work, so you generally don't see them explained in guides.
Furthermore, rulers are a standard feature of every image editor and layout application, so most people have used them before and rely on them being there, for detailed layout work.

I fully support the suggestion that the editor remembers your choice of disabling them, but having them disabled by default, when they are an expected feature for layout work, would be like having scrollbars disabled by default, because you happen to never work with scenes that don't fit on your screen.
